src/handlers/decision.rs

```diff
@@ -152,6 +164,164 @@ pub(super) async fn handle_command(
     }
 }
 
+fn build_status_comment(
+    history: &BTreeMap<String, Vec<UserStatus>>,
+    current: &BTreeMap<String, Option<UserStatus>>,
+) -> anyhow::Result<String> {
+    let mut comment = "| Team member | State |\n|-------------|-------|".to_owned();
+    for (user, statuses) in history {
+        let mut user_statuses = format!("\n| {} |", user);
+
+        // previous stasuses
+        for status in statuses {
+            let status_item = format!(" ~~{}~~ ", status.resolution.to_string());
+            user_statuses.push_str(&status_item);
+        }
+
+        // current status
+        let user_resolution = match current.get(user) {
+            Some(current_status) => {
+                if let Some(status) = current_status {
+                    format!("**{}**", status.resolution.to_string())
+                } else {
+                    "".to_string()
+                }
+            }
+            None => bail!("user {} not present in current statuses list", user),
```

Comment:
Shorthand for:
`return Err(anyhow!("user {} not present in current statuses list", user));`

Reply:
In the future, we can discuss what we do for such cases, or if we'll ever support having someone in the current status but not in history, etc. I wouldn't decide now.